projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/BaseDataJobDeploymentCrudIT.java

```diff
@@ -109,7 +111,7 @@ public void testDataJobDeploymentCrud() throws Exception {
     Assertions.assertFalse(StringUtils.isBlank(testJobVersionSha));
 
     // Setup
-    String dataJobDeploymentRequestBody = getDataJobDeploymentRequestBody(testJobVersionSha);
+    String dataJobDeploymentRequestBody = getDataJobDeploymentRequestBody(testJobVersionSha, "3.9");
```

Comment:
Make sure that the test verifies all fields through the controller/API after each deployment change.

Reply:
I still don't see this verification. For example, the test deploys a job with specific resources, and then doesn't even check them.